server2/Arcadia.Assistant/Arcadia.Assistant.Sharepoint/SharepointItemSynchronization.cs

```diff
@@ -14,19 +14,67 @@
 
     using Microsoft.Extensions.Logging;
 
-    public abstract class SharepointItemSynchronization<T>
+    public abstract class SharepointItemSynchronizationBase
     {
-        #region ctor
+#pragma warning disable CS8618 // Non-nullable field is uninitialized. Consider declaring as nullable.
```

Comment:
????? try calling it when it's null, good luck. should be declared as nullable. Why do you need events all of a sudden?

Reply:
Removed